test/features/step_definitions/sdk_steps.rb

```diff
@@ -17,3 +18,20 @@
     When I click on #{action_button} ()
   }
 end
+
+element = '(\w+(?:|\(.*\)) \(\w*\)(?:| index \d+))'
```

Comment:
Why is this needed?

Reply:
It's a regex from the monster gem used to identify HTML elements. It's used in line 24.

For the tests I will move the "open new tab" step to the monster gem so we can reuse the regex there.

Reply:
Ok I see, thanks